pt/lessons/advanced/erlang.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: page
+title: Interoperabilidade com Erlang
+category: advanced
+order: 1
+lang: pt
+---
+
+Um dos benefícios adicionais em se construir em cima da ErlangVM é a abundância de bibliotecas existentes disponíveis para nós. A interoperabilidade nos permite usar essas bibliotecas e a biblioteca padrão Erlang a partir do nosso código Elixir. Nessa lição, nós vamos olhar como acessar funcionalidades da biblioteca padrão juntamente com pacotes Erlang de terceiros.
+
+## Sumário
+
+- [Biblioteca padrão](#biblioteca-padrao)
+- [Pacotes Erlang](#pacotes-erlang)
+- [Diferenças notáveis](#diferencas-notaveis)
+  - [Atoms](#atoms)
+  - [Strings](#strings)
+  - [Variáveis](#variaveis)
+
+
+## Biblioteca padrão
```

Comment:
You should use `## <a name="biblioteca-padrao"></a>Biblioteca padrão` when the title has accentuation.